### ISSUE TITLE
求问如何选择的val数据集,有什么特殊的标准吗

### ISSUE THREAD

**Issue by Amelie01** [leeyeehoo/CSRNet-pytorch#21]

**Comment by neverland0621** [comment]
+1shanghaiitech这个数据集里只有训练姐和测试集，验证集是怎么选取出的呢？

**Comment by ZhengPeng7** [comment]
You can just randomly selected about 10 or 20 images from the training set as validation set. But in my personal view, I prefer to train also on the validation set since the training set is too small.

**Comment by visionxyz** [comment]
理论上要有一个验证集，但是这个数据集太小了，验证集的选择会让你的model相差比较大。然后不同人的不同验证集也会让结果难以复现。最后非常多的论文不明说这个选择的问题。
Normally, we should select some images as validation set, we can chose 10% images randomly. But since this dataset is too small, different validation sets will lead to different result. Some results will be very bad if your validation set is not good. And since we chose them randomly, it is hard to reporoduce others' results. A lot of papers don't mention it very clearly.
